packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap

```diff
@@ -0,0 +1,1034 @@
+              "id": "2",
+              "image": "https://picsum.photos/600/400/?random",
+              "key": "church",
+              "overlayColor": Array [
+                "#6BAC43",
+                "#6BAC43",
+              ],
+              "title": "My Church",
+            },
+            Object {
+              "description": "Pray for the people at your campus",
+              "id": "3",
+              "image": "https://picsum.photos/600/400/?random",
+              "key": "campus",
+              "overlayColor": Array [
+                "#6BAC43",
+                "#6BAC43",
+              ],
+              "title": "My Campus",
+            },
+            Object {
+              "description": "Pray for those people in your community",
+              "id": "4",
+              "image": "https://picsum.photos/600/400/?random",
+              "key": "community",
+              "overlayColor": Array [
+                "#6BAC43",
+                "#6BAC43",
+              ],
+              "title": "My Community",
+            },
+            Object {
+              "component": <UserPrayerList />,
+              "description": "Revisit your prayers",
+              "id": "5",
+              "image": "https://picsum.photos/600/400/?random",
+              "key": "prayers",
+              "overlayColor": Array [
+                "#6BAC43",
+                "#6BAC43",
+              ],
+              "title": "My Prayers",
+            },
+          ]
+        }
+        decelerationRate="fast"
+        disableVirtualization={false}
+        getItem={[Function]}
+        getItemCount={[Function]}
+        horizontal={true}
+        initialNumToRender={10}
+        initialScrollIndex={0}
+        keyExtractor={[Function]}
+        loadingStateObject={
+          Object {
+            "node": Object {
+              "id": "fakeId0",
+              "isLoading": true,
+              "title": "",
+            },
+          }
+        }
+        maxToRenderPerBatch={10}
+        numColumns={1}
+        onContentSizeChange={[Function]}
+        onEndReachedThreshold={2}
+        onLayout={[Function]}
+        onMomentumScrollEnd={[Function]}
+        onScroll={[Function]}
+        onScrollBeginDrag={[Function]}
+        onScrollEndDrag={[Function]}
+        refreshing={false}
+        renderItem={[Function]}
+        scrollEventThrottle={50}
+        showsHorizontalScrollIndicator={false}
+        snapToAlignment="start"
+        snapToInterval={512}
+        stickyHeaderIndices={Array []}
+        style={
+          Object {
+            "paddingLeft": 16,
+          }
+        }
+        updateCellsBatchingPeriod={50}
+        viewabilityConfigCallbackPairs={Array []}
+        windowSize={21}
+      >
+        <View>
+          <View
+            onLayout={[Function]}
+            style={
+              Array [
+                Object {
+                  "flexDirection": "row",
+                },
+                null,
```

Comment:
Is this `null` intended?

Reply:
According to the tests in TAP on the discover tab, this `null` is also fine. Whew!